app/src/main/res/values/strings.xml

```diff
@@ -899,8 +899,7 @@
 
     <string name="save_csv_data">Save CSV Data</string>
     <string name="view_map">View Map</string>
-    <string name="data_recording_start">Data recording started</string>
-    <string name="data_recording_stop">Data recording stopped</string>
+    <string name="data_recording_start">Data Recording: Started</string>
```

Comment:
Can we implement a similar colon for Data recording stop?

Reply:
Actualluy instead of data recording stopped it shows file stored location snackbar